UI_UX/Essential_Screens/Calendar_Screen/Login_Page/documentation.md

```diff
@@ -0,0 +1,83 @@
+# Login and Signup Screen
+
+## Preview of the login and sign up screens :
+
+![figma11](https://user-images.githubusercontent.com/56999749/133961011-3ffe38db-f93d-440c-b930-05e1abd664f0.JPG)
+
+<br>
+
+1. First we need to add frames on the figma screen.
+<br>
+
+- To add frames, select region tools from the navbar of figma screen.
+
+![figma1](https://user-images.githubusercontent.com/56999749/133961023-e20d6234-94a7-42b9-a356-3ed54e3e12f7.JPG)
+
+2. We can then use shape tools on the frame to adjust its size.
+<br>
+
+- To use shape tools -> Search for the rectangle option on the navbar and select any shape.
+
+3. We can then change the background colour according to our choices.
+<br>
+
+- To change the background colour -> Select the frame -> Look for fill option in the Design tab and then select the colour.
+
+![figma2](https://user-images.githubusercontent.com/56999749/133961170-1f9ca4dd-f5cb-4071-ba33-ee0e86093fd9.JPG)
+
+<br>
+
+- We can also apply gradient effect on the background.
+<br>
+To apply gradient effect -> Click on Solid in the fill tab and then select any effect -> Adjust the colour according to your choice.
+
+![figma3](https://user-images.githubusercontent.com/56999749/133961179-be1da967-009a-448f-80e5-e775d9b6f410.JPG)
+
+4. Then we can add shape tools, pictures and text to the landing screen in order to beautify it.
+
+![figma4](https://user-images.githubusercontent.com/56999749/133961335-a4885d91-c58d-450e-91ab-21cf7c59b1d4.JPG)
+
+<br>
+
+- To add pictures -> We can select any picture from the internet and then copy and paste it in the desired place.
+
+- To add text -> Search for the text option in the navbar.
+
+![figma5](https://user-images.githubusercontent.com/56999749/133961343-e3ea4e1b-93fa-4d53-abf8-952e8482a1ad.JPG)
+
+5. For the buttons on the landing page we need to add two rectangle from the shape tools option. 
```

Comment:
Here, I think you missed few screens. 
1. Show the two blank rectangles for buttons.
2. Add the label over it.